java/com/google/turbine/parse/Token.java

```diff
@@ -23,8 +23,8 @@
   RPAREN(")"),
   LBRACE("{"),
   RBRACE("}"),
-  LBRACK("<"),
-  RBRACK(">"),
+  LBRACK("["),
+  RBRACK("]"),
```

Comment:
@cushon I wasn't sure about this change. Do you know why the value was off/wrong?

Reply:
Huh, nice catch. I think these strings are just mostly used for debugging, and this was an oversight that went unnoticed